index.html

```diff
@@ -12,29 +12,66 @@
 	</head>
 	<body>
 		<h1>BATTLESHIP</h1>
-		<div class="board-container">
-			<input type="text" class="player-name" value="Player 1">
-			<table class="board" id="board0"></table>
-		</div>
-		<div class="board-container">
-			<input type="text" class="player-name" value="Player 2">
-			<table class="board" id="board1"></table>
-		</div>
-		<div id="message"></div>
-		<input type="button" id="switch-turn" value="Switch turn">
 
-		<div class="option">
+		<!-- Probably add CSS to this -->
+		<div style="text-align:center" id="menu">
+            <h2>WELCOME</h2>
+            <p>Please enter player's names: </p>
+            <form action="">
```

Comment:
The form tag is only needed when submitting to the server, so it can be removed.